python/pyspark/sql/worker/create_data_source.py

```diff
@@ -127,31 +127,19 @@ def main(infile: IO, outfile: IO) -> None:
             options[key] = value
 
         # Instantiate a data source.
-        try:
-            data_source = data_source_cls(options=options)  # type: ignore
-        except Exception as e:
-            raise PySparkRuntimeError(
-                errorClass="DATA_SOURCE_CREATE_ERROR",
```

Comment:
Is the error condition `DATA_SOURCE_CREATE_ERROR ` used somewhere else? If not, please, remove it.